### ISSUE TITLE
Single quote syntax highlighting in Playground

### ISSUE THREAD

**Issue by nicot** [PRQL/prql#1932]
Single quote syntax highlighting isn't working the same way double quote syntax highlighting works in the playground:

<img width="286" alt="Screenshot 2023-02-22 at 9 45 02 AM" src="https://user-images.githubusercontent.com/3722365/220697442-19d4ced8-f7a8-41f3-803e-7d9bade32f3b.png">

Single quotes don't mess up parsing though. I think that the double quote syntax highlighting is a bit better, and it'd be great to swap the single quote highlighting to be the same as the double quote one.

**Comment by max-sixty** [comment]
Yes agree, the playground syntax highlighting could do with some work. Contributions very welcome — the file is at ﻿https://github.com/PRQL/prql/blob/14f6e7180cd8e10ddea8ecbae731b044fdd758d6/playground/src/workbench/prql-syntax.js

**Comment by vanillajonathan** [comment]
Another question is whether single quote syntax should be allowed?

The disadvantage is that people write queries differently and then there are disagreements and bickering about style, and more to be covered by style guides and best practices. An alternative would be to only support double quotes.

**Comment by max-sixty** [comment]
@vanillajonathan we'd need a strong reason to change the current approach of allowing both. I don't think we have a good reason to change.

(and we should have that discussion on a separate issue)
